clients/drcachesim/tracer/raw2trace.h

```diff
@@ -388,6 +391,22 @@ struct trace_metadata_reader_t {
     check_entry_thread_start(const offline_entry_t *entry);
 };
 
+#ifdef BUILD_PT_POST_PROCESSOR
+/**
```

Comment:
If a struct is Doxygen-documented it should have Doxygen comments on every field, not just the struct itself.  However, I don't think it needs any docs.  raw2trace used to be split into pieces and one was exported for use externally and that is why it has some Doxygen-documented interfaces: probably those should have been changed to regular comments when that split and that exposed interface was refactored and removed recently.